src/rgw/rgw_auth_keystone.cc

```diff
@@ -129,6 +129,7 @@ TokenEngine::get_creds_info(const TokenEngine::token_envelope_t& token,
     rgw_user(token.get_project_id()),
     /* User's display name (aka real name). */
     token.get_project_name(),
+    token.get_user_name(),
```

Comment:
How about the perm_mask here? It returns RGW_PERM_FULL_CONTROL here, will the subuser also have RGW_PERM_FULL_CONTROL? @cbodley @mattbenjamin : Radoslaw has left a comment that 'Keystone doesn't support RGW's subuser concept' (see below) Any idea why?

Reply:
Yes because in Keystone response we don't have any field that represent permission for users so we give `RGW_PERM_FULL_CONTROL` to all keystone subusers and [here](https://github.com/ceph/ceph/pull/33210/files?file-filters%5B%5D=.cc#diff-15b9f415d50309d52c852a37f9f97705R535) we set the AuthInfo perm_mask to RGWSubUser perm_mask.